src/timeMachine/components/TimeMachineFluxEditor.tsx

```diff
@@ -102,15 +102,16 @@ const TimeMachineFluxEditor: FC = () => {
       row = currentRange.endLineNumber + 1
     }
 
-    let text = ''
+    let text = !!(func as FluxToolbarFunction).example
+      ? (func as FluxToolbarFunction).example
```

Comment:
this is a temporary fix until we parse flux signature from the flux type and name properties. Prior this 'fix', injecting a function (with dynamic flux feature flag on) returns undefined. at least now it will say flux.name()